### ISSUE TITLE
♿ Enable keyboard navigation in the navigation drawer

### ISSUE THREAD

**Issue by vnys** [equinor/design-system#895]
The navigation drawer in the Storefront can’t be navigated using a keyboard, because the list items lack `tabindex` attributes and keypress event listeners.

**Comment by mimarz** [comment]
This will be solved by #164
